.github/workflows/build.yml

```diff
@@ -26,9 +26,18 @@ jobs:
           npm run prepare
         env:
           CI: true
-      - name: npm build, test
+      - name: npm build
         run: |
           npm run build
+        env:
```

Comment:
FYI, `CI` is always set to `true`: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables